package.json

```diff
@@ -7,6 +7,9 @@
     "npm": "please-use-yarn",
     "yarn": ">= 1.22.0"
   },
+  "workspaces": [
```

Comment:
Why we need it here? Everything is installed in the root.

Reply:
`Nx` installs everything in the root, but doesn't validate the versions installed in the root and the one defined in the library `package.json`, so `yarn workspaces` fixes this installing the dependencies from the library instead, as we did in `spectator`, it's helpful here too.